### PR TITLE
Add a pizza trivia (pizza patatine)

### PR DESCRIPTION
**Pull request by Pondy007**
## Description

This PR adds a pizza trivia.

## What type of PR is this? (check all applicable)

- [x] ☝️ Add a pizza fact or trivia
- [ ] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [ ] 📝 Documentation Update

## Related Tickets & Documents

Relates to #36 

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

## [optional] What GIF best describes this PR or how it makes you feel?
image

### CODE REVIEW

File: pizza-facts-&-trivia.md [hunk 1]
```diff
@@ -45,3 +45,5 @@ Delight in intriguing and lesser-known facts that will surprise and entertain yo
   - 100 acres
 - What unusual pizza topping is popular in Japan?
   - Mayonnaise
+- What unusual pizza topping can be found in France, Belgium or Italy ?
+  - In some European countries, especially France, Belgium or Italy, (and maybe elsewhere in the world) some people like to add french fries as topping for pizzas. One call it Pizza Patatine (or Pizza Patatosa) and Domino's already launched for a limited time such pizzas (in 2023 in France).
```
Comment (adiati98) [inline pizza-facts-&-trivia.md:49]: Just minor wording and punctuation fixes here.

```suggestion
  - In some European countries, especially France, Belgium, or Italy (and maybe elsewhere in the world), some people like to add french fries as topping for pizzas. One call it Pizza Patatine (or Pizza Patatosa), and Domino's already launched such pizzas for a limited time (in 2023 in France).
```

**Review by adiati98 — COMMENTED** [review]
Hi @Pondy007 ,
Thank you for your PR! 🙌

I left a review here for you to fix before we merge your PR.
Let me know when it's fixed and ready for a review.
Thank you. 🙂

**Comment by Pondy007** [comment]
Hi @adiati98 ,
That's fine for me. Thanks.
Best regards and have a nice day !

**Comment by adiati98** [comment]
Thanks for the changes and for adding one great pizza trivia, @Pondy007 ! 😄
I just knew that some pizzas have french fries as topping. 😂

git merge GIF

**Review by adiati98 — APPROVED** [review]

File: pizza-facts-&-trivia.md [hunk 2]
```diff
@@ -45,3 +45,5 @@ Delight in intriguing and lesser-known facts that will surprise and entertain yo
   - 100 acres
 - What unusual pizza topping is popular in Japan?
   - Mayonnaise
+- What unusual pizza topping can be found in France, Belgium or Italy ?
```
Comment (BekahHW) [inline pizza-facts-&-trivia.md:48]: This is a great fact! There's an extra space before the ? : `Italy ?` Would you mind removing it and we'll get it merged in?

**Review by BekahHW — CHANGES_REQUESTED** [review]

File: pizza-facts-&-trivia.md (hunk 2 above)
Comment (Pondy007) [reply pizza-facts-&-trivia.md:48]: Hi @BekahHW , you are right ! I will correct.
For your information, in France, we do add a space before any mark (!, ?, : ...).
Have a nice day !

Comment (BekahHW) [reply pizza-facts-&-trivia.md:48]: @Pondy007 that's interesting! Thanks for letting me know.

**Review by BekahHW — APPROVED** [review]

**Pull request merged**